docs/sphinx/source/contributing.rst

```diff
@@ -220,5 +319,5 @@ This documentation
 
 If this documentation is unclear, help us improve it! Consider looking
 at the `pandas
-documentation <http://pandas.pydata.org/pandas-docs/version/0.18.1/
+documentation <http://pandas.pydata.org/pandas-docs/stable/
```

Comment:
What is the difference between "stable" and "latest"? It seems that sometimes "latest" refers to master, but other times to the latest tag, which might not actually be the highest tag. (For example, Docker has some subtleties here, if I recall correctly.)

Reply:
most python projects configure readthedocs such that stable is the most recently released version (git tag), whereas latest uses the current state of the master branch.